### DISCUSSION TITLE
Where are the traces/span stored in docker container?

### DISCUSSION THREAD

**Discussion by ananthanandanan** [Arize-ai/phoenix#2333] [General]
I had set up the docker container for Phoenix locally and I was able to run my application and log the traces. But I was curious to know where they are stored, since when I rebuild the container the traces are gone and If they are stored locally I can use a volume mount for the time being to keep the data persistent.

**Comment by RogerHYang** [comment]
Hi @ananthanandanan. Persistence and storage is on our [roadmap](https://github.com/Arize-ai/phoenix/issues/1689) and we plan to work on that very soon. Right now the traces are stored in memory. For the time being, you can extract the data using px.Client. See our docs [here](https://app.gitbook.com/o/-MB4weB2E-qpBe07nmSL/s/ShR775Rt7OzHRfy5j2Ks/api/client#usage) for details.

**Reply by ananthanandanan** [reply]
Thanks, I am looking forward to the feature of persistent storage for Phoenix.
